charts/scalardl-audit/values.yaml

```
@@ -260,3 +260,9 @@ auditor:
 
   # -- Name of existing secret to use for storing database username and password
   existingSecret: ""
+
+  serviceAccount:
+    # -- Name of the existing service account resource
+    serviceAccountName: ""
+    # -- Specify to mount secret or not
```

```suggestion
    # -- Specify to mount a service account token or not
```
Should it be like this for consistency?

---

Thank you for your suggestion!
I think it is better than the current description.
So, I will apply it. Thank you!